examples/jsm/postprocessing/EffectComposer.js

```diff
@@ -62,6 +62,7 @@ class EffectComposer {
 		}
 
 		this.copyPass = new ShaderPass( CopyShader );
```

Comment:
Can these (including two checks) lines be changed to just `this.copyPass = new CopyPass()`?

Reply:
This would break app code which uses `examples/js` version of `EffectComposer` since the dependencies would be different.